### DISCUSSION TITLE
Colors in the multi-variant display

### DISCUSSION THREAD

**Discussion by lidijaberke** [GMOD/jbrowse-components#4715] [General]
Hi! 

The new multi-variant display looks wonderful, thank you! I saw that mouseover is still to be added, but in the mean time, is there a color legend for the variants (blue, green, grey and light blue) available somewhere in the documentation?

Cheers,
Lidija

**Comment by cmdcolin** [comment]
Thanks for the kind words!

Indeed better mouseover and legend and docs are needed, but the color scheme currently is

grey: homozrgous REF
blue: homozygous ALT
light blue/teal: heterozygous ALT
purple/periwinkle: everything else, this is probably pending a change though

I am considering making this all configurable too
